packages/module/src/SkeletonTable/SkeletonTable.tsx

```diff
@@ -3,14 +3,16 @@ import { Caption, Table, TableProps, TableVariant, Tbody, Td, Th, Thead, Tr } fr
 import { Skeleton } from '@patternfly/react-core';
 
 export type SkeletonTableProps = Omit<TableProps, 'ref'> & {
-  /** Indicates the table variant */
   variant?: TableVariant;
-  /** The number of rows the skeleton table should contain */
   rows?: number;
-  /** Any captions that should be added to the table */
   caption?: ReactNode;
-  /** Custom OUIA ID */
   ouiaId?: string | number;
+  isSelectable?: boolean;
+  isExpandable?: boolean;
+  sortBy?: {
+    index: number;
+    direction: 'asc' | 'desc';
+  };
```

Comment:
Please do not remove the comments - these prop descriptions are displayed in the docs. Can you please add them also for the new props?